TESTPLAN.md

```diff
@@ -19,93 +20,65 @@
 
 1.  BUILDABLE.  It always builds.
 
-    `npm run build` makes an addon:
-
     ```
-    -rw-r--r--   1 glind  staff     13 Nov  8 15:36 .gitignore
-    lrwxr-xr-x   1 glind  staff     43 Dec  8 13:09 linked-addon.xpi@ -> pug.experience@shield.mozilla.org-1.0.0.xpi
-    -rw-r--r--   1 glind  staff  21975 Dec  8 13:09 pug.experience@shield.mozilla.org-1.0.0.xpi
+    cd addon/webextension
+    web-ext lint
+    web-ext run
     ```
 
 2.  LINTABLE:  It lints, with **warnings** but **no errors**.
 
     `npm run eslint`
 
 
-3.  INSTALLABLE:  Full Embedded WebExtension INSTALLS (on nightly)
-
-    `npm run firefox`
-
-
-4.  (WebExtenion also works)
+3.  RUNNABLE
 
+    ```
+    cd addon/webextension
+    web-ext lint
+    web-ext run
+    ```
 
 
 ## Tests:
 
-1. `npm run firefox` (make sure this launches Firefox Nightly until the add-on is signed, at which point you can test on Release, Developer Edition/Aurora, and Nightly).
-
-1. **Preference Exists** (as false)
 
-    - Open `about:config`
-    - verify the `extensions.pug.lookingglass`
-    - Boolean config is set to `false`.
-
-1. **Header sending depends on preference**
-
-    Goal: convince that the HEADER `X-1057` will only be sent if the preference is on.
+1. **Header ALWAYS sends on special sites.**
 
+    Goal: convince that the HEADER `X-1057` will be sent.
 
     1. Setup
 
         - open https://www.whatismybrowser.com/detect/what-http-headers-is-my-browser-sending
-        - open `about:config`
-        - PREFERENCE:  `extensions.pug.lookingglass`
-
-    1. Preference FALSE :: no `X_1057` header.
-        - refresh https://www.whatismybrowser.com/detect/what-http-headers-is-my-browser-sending
-        - Confirm that `X_1057` is NOT in the list
-        - Confirm that `X_1057` is NOT in the list
 
 
-    2. Preference TRUE :: yes `X_1057` header.
+    2. Always yes `X_1057` header.
         - set PREFERENCE to `true`
         - refresh https://www.whatismybrowser.com/detect/what-http-headers-is-my-browser-sending
         - Confirm that `X_1057` `true` IS in the list of headers
 
-    3.  Turn off the pref, and confirm headers stop sending.
-
     Notes:
 
-    1. IFF you are running from `web-ext`, it will behave as though the pref is true.
-    2. We don't have a good 'un-test' for showing the header isn't sent *everywhere*.  It's contolled by `addon/webextension/{manifest.json,background.js}`
+    1. We don't have a good 'un-test' for showing the header isn't sent *everywhere*.  It's contolled by `addon/webextension/{manifest.json,background.js}`
```

Comment:
They could pick a random page and inspect the request headers for the first GET request (requesting the page) in the Network tab in devtools (may be too involved); this would only show one instance of it NOT being there.

Reply:
+1